### PR TITLE
chore(docs): add v2.11.0 released changelog

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

This PR updates the v2.11.0 changelogs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+v2.11.0-RC2 / 2021-07-13
```
Comment (prateekpandey14) [inline CHANGELOG.md:1]: Add the GA tag as well

Comment (kmova) [inline CHANGELOG.md:1]:
```suggestion
v2.11.0 / 2021-07-15
========================
* fix(status): fetch volume status using controller podIP (#112, @shubham14bajpai)
* fix(csi): prevent volume mount on multiple nodes simultaneously (#107, @shubham14bajpai)

v2.11.0-RC2 / 2021-07-13
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+========================
```
Comment (kmova) [inline CHANGELOG.md:2]:
```suggestion
========================
* fix(status): fetch volume status using controller podIP ([#112](https://github.com/openebs/jiva-operator/pull/112),[@shubham14bajpai](https://github.com/shubham14bajpai))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+
+
+v2.11.0-RC1 / 2021-07-07
+========================
+* fix(status): fetch volume status using controller podIP ([#112](https://github.com/openebs/jiva-operator/pull/112),[@shubham14bajpai](https://github.com/shubham14bajpai))
```
Comment (kmova) [inline CHANGELOG.md:7]:
```suggestion
```

**Pull request merged**